lib/rollbax.ex

```diff
@@ -124,7 +125,12 @@ defmodule Rollbax do
   end
 
   defp get_config(key, default) do
-    Application.get_env(:rollbax, key, default)
+    case Application.get_env(:rollbax, key, default) do
+      {:system, var} ->
+        System.get_env(var) || raise ArgumentError "environment variable #{var} is not set"
```

Comment:
Using `#{var}` will crash if `var` is not a binary. Let's use `#{inspect(var)}`.

Also, the error message could be improved a bit, maybe by mentioning `"*system* environment variable #{inspect(var)} is not set"` so that it's not confused with application environment.

Wdyt?